src/main/resources/messages_en.properties

```diff
@@ -4,7 +4,6 @@ mail.OM_CONTROL_EMPLOYEES_CONTENT.subject=Reminder: Control employees
 mail.OM_RELEASE.subject=Reminder: Perform release
 mail.OM_ADMINISTRATIVE.subject=Reminder: Administrative
 mail.OM_SALARY.subject=Reminder: Salaries
```

Comment:
Wenn wir schon dabei sind, aufzuräumen: bei den messages_en fehlen ein paar Templates, die in Deutsch verfügbar sind => bitte ergänzen